cfgov/unprocessed/css/organisms/footer.less

```diff
@@ -1,19 +1,4 @@
-@import (less) '../enhancements/typography.less';
-
-/* ==========================================================================
-   consumerfinance.gov
-   footer
-   ========================================================================== */
-
-/* topdoc
-  name: CF.gov site-wide footer.
-  family: cfgov-organisms
-  patterns:
-    - notes:
-        - "For the markup please see _layouts/organisms/footer.html."
-  tags:
-    - cfgov-organisms
-*/
+@import (reference) '../enhancements/typography.less';
```

Comment:
I'm not sure this can be `reference` with the mixin references. E.g. does `.m-list__links();` actually resolve in the generated code?

Reply:
Yep. The only time it doesn't actually work is referencing nested mixins that are defined with the `&__whatever` pattern.